include/eve/module/core/regular/if_else.hpp

```diff
@@ -49,19 +48,17 @@ namespace eve
 //!
 //!     The following calls where `x`, `y` and `z` are values can be optimized:
 //!
-//!        * `if_else(x, y, allbits< T >()) ` writing: `if_else(x, y, eve::allbits_)`
-//!        * `if_else(x, y, one< T >()    ) ` writing: `if_else(x, y, eve::one_    )`
-//!        * `if_else(x, y, mone< T >()   ) ` writing: `if_else(x, y, eve::mone_   )`
-//!        * `if_else(x, y, zero< T >()   ) ` writing: `if_else(x, y, eve::zero_   )`
-//!        * `if_else(x, allbits< T >(), z) ` writing: `if_else(x, eve::allbits_, z)`
-//!        * `if_else(x, one< T >(), z    ) ` writing: `if_else(x, eve::one_, z    )`
-//!        * `if_else(x, mone< T >(), z   ) ` writing: `if_else(x, eve::mone_, z   )`
-//!        * `if_else(x, zero< T >(), z   ) ` writing: `if_else(x, eve::zero_, z   )`
+//!        * `if_else(x, y, allbits< T >())` writing: `if_else(x, y, allbits)`
```

Comment:
valmin and valmax are also supproted when usinged, I think we need anotehr lsit for unsigned type